frontend/src/component/events/EventDiff/EventDiff.tsx

```diff
@@ -96,14 +96,14 @@ const EventDiff = ({
         };
     };
 
-    let changes;
+    let changes: any[] = [];
 
     if (diffs) {
         changes = diffs
             .map(buildDiff)
             .sort(sort)
             .map(({ value }) => value);
-    } else {
+    } else if (entry.data == null || entry.preData == null) {
```

Comment:
you have to expand a few lines below to see how changes.length was never 0

Reply:
the entire new or old code should be shown only when one of the data or preData is null o rundefined. If both are provided we should not go in here